.github/fabricbot.json

```diff
@@ -1923,6 +1923,422 @@
           }
         ]
       }
+    },
+    {
+      "taskType": "trigger",
+      "capabilityId": "IssueResponder",
+      "subCapability": "PullRequestResponder",
+      "version": "1.0",
+      "config": {
+        "conditions": {
+          "operator": "and",
+          "operands": [
+            {
+              "operator": "or",
+              "operands": [
+                {
+                  "name": "isAction",
+                  "parameters": {
+                    "action": "opened"
+                  }
+                },
+                {
+                  "name": "isAction",
+                  "parameters": {
+                    "action": "reopened"
+                  }
+                }
+              ]
+            },
+            {
+              "name": "activitySenderHasPermissions",
+              "parameters": {
+                "permissions": "read"
+              }
+            },
+            {
+              "operator": "not",
+              "operands": [
+                {
+                  "name": "isActivitySender",
+                  "parameters": {
+                    "user": "nuget-client "
+                  }
+                }
+              ]
+            },
+            {
+              "operator": "not",
+              "operands": [
+                {
+                  "name": "hasLabel",
+                  "parameters": {
+                    "label": "Community"
+                  }
+                }
+              ]
+            }
+          ]
+        },
+        "eventType": "pull_request",
+        "eventNames": [
+          "pull_request",
+          "issues",
+          "project_card"
+        ],
+        "taskName": "[Label community PRs] [1-1] Label the community PRs",
+        "actions": [
+          {
+            "name": "addLabel",
+            "parameters": {
+              "label": "Community"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "taskType": "scheduled",
```

Comment:
Shouldn't this amend the already existing tasks that close stale PRs?

Reply:
Thanks for reviewing!
The existing tasks are for non-community PRs (exclude PRs with community label). That's why all community PRs have no stale process currently. The bots stale non-community PRs in 30 days and close them in another 15 days if no activity.
The newly added tasks are for community PRs only. They stale PRs in 30 days and close them in another 330 days if no activity.